envi/archs/arm/const.py

```diff
@@ -347,11 +354,16 @@ def instrenc(encoding, index):
 INS_ASR = 85
 INS_ROR = 85
 INS_RRX = 85
-#New commands for ARMV7
-INS_CLREX = 85
-INS_DMB = 85
-INS_DSB = 85
-INS_ISB = 85
+#New commands for ARMV7 - need to be fitted too.
+INS_DBG = 85
+INS_BF  = 85
+INS_CLREX   = IENC_UNCOND_PLD #FIXME
```

Comment:
we really need to make these all their own number.  we'll probably do this in const.py and just have a list of names and iterate through them to generate their numbers sequentially.